### ISSUE TITLE
Automatically create submodule commits for master repo from language repo commits

### ISSUE THREAD

**Issue by stonemaster** [dlang-tour/bot#3]
This is rather a question whether this might be feasible to improve the overall process: Is it possible that the bot watches for commits in the language repositories and creates a new commit to update the languages submodule in the core repository to that commit? As asked in https://github.com/stonemaster/dlang-tour/issues/482 the problem is that the core repo doesn't contain the most recent translations. This isn't a problem for Travis because here we have magic that automatically updates the submodules to the according HEAD of the language repos. But it is a problem when someone runs the tour offline using dub. 
Or generally if someone wants to develop locally the translation submodules need be updated manually which isn't optimal in my eyes.

**Comment by wilzbach** [comment]
> Is it possible that the bot watches for commits in the language repositories and creates a new commit to update the languages submodule in the core repository to that commit? 

Yep. We could give the bot "write" access to the base repository, s.t. it automatically makes the update commit.
It would be something like:
- clone the base repository (needs to be on `/tmp` at Heroku)
- do `git submodule foreach git pull origin master`
- do a commit (git doesn't allow empty commits anyways)
- push the commit with the bot's credentials

> But it is a problem when someone runs the tour offline using dub.

Yeah, but we first need to figure out how we can tell `dub` to clone the submodules.

> Or generally if someone wants to develop locally the translation submodules need be updated manually which isn't optimal in my eyes.

For local development there's the `--lang-dir` flag which allows to run the tour with any language folder.
